data/build.gradle

```diff
@@ -0,0 +1,62 @@
+
+apply plugin: 'com.android.library'
+//apply plugin: 'kotlin'
```

Comment:
remove this line.
why you commit this "commented line" to the origin repository?
before commit, you have to review your own commit again.